File: 백준/Silver/2563. 색종이/색종이.py
white_board = [[0] * 100 for _ in range(100)]
result = 0

n = int(input())

for _ in range(n):
    x, y = map(int, input().split(' '))
    for i in range(10):
        for j in range(10):
            white_board[x+i][y+j] = 1

for i in range(100):
    for j in range(100):
        if white_board[i][j] == 1:
            result +=1

print(result)

File: 백준/Silver/2563. 색종이/색종이.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        InputStreamReader reader = new InputStreamReader(System.in);
        BufferedReader br = new BufferedReader(reader);
        
        boolean[][] matrix = new boolean[100][100];
        
        int N = Integer.parseInt(br.readLine());
        
        for(int i=0; i<N; i++) {
            String[] XY = br.readLine().split(" ");
            int X = Integer.parseInt(XY[0]) - 1;
            int Y = Integer.parseInt(XY[1]) - 1;
            
            for(int j=0; j<10; j++) {
                for(int k=0; k<10; k++) {
                    matrix[X+j][Y+k] = true;
                }
            }
        }
        
        int result = 0;
        
        for(int x=0; x<100; x++) {
            for(int y=0; y<100; y++) {
                if(matrix[x][y]) {
                    result += 1;
                }
            }
        }
        
        System.out.println(result);
    }
}